### ISSUE TITLE
PrometheusRules provided by Helm chart, but not included in manifests

### ISSUE THREAD

**Issue by tylerauerbeck** [metallb/metallb#1660]
When taking a look at https://github.com/metallb/metallb/issues/1019 , I noticed that default PrometheusRules are provided by the Helm chart, they are not present in the static manifests. Not sure if there is a historical reason for that or not, but it seems to me that PrometheusRules would be valuable regardless of how the app is deployed.

**Comment by fedepaol** [comment]
Yep, the problem is the manifests do not offer the same flexibility in terms of configuratoin that helm offers (and in this case even going with kustomize is an overkill).
What we can do is either provide an example on how to set custom rules, or to point the users to the helm ones so they can craft their own.
